tools/WinObjC.Packaging/WinObjC.Packageable.targets

```diff
@@ -78,7 +78,8 @@
         <PackagePath>$(PackageResourcesBasePath)\%(Filename)%(Extension)</PackagePath>
       </PackageFile>
       <!-- Package up .pri files associated with any .winmds -->
-      <PackageFile Include="$(ProjectPriFullPath)" Condition="Exists('$(ProjectPriFullPath)') and Exists('@(WinMDFullPath)')">
+      <_PriFileNames Include="$(ProjectPriFullPath)" />
+      <PackageFile Include="@(_PriFileNames)" Condition="Exists('%(FullPath)') and Exists('$(_WinMDFilePath)')">
```

Comment:
I was talking about `_WinMDFilePath`; it didn't look like it was introduced in this pull request